lib/awful/tag.lua

```diff
@@ -309,7 +309,7 @@ function tag.object.delete(self, fallback_tag, force)
     end
 
     -- If no tags are visible, try and view one.
-    if target_scr.selected_tag == nil and ntags > 0 then
+    if target_scr.selected_tag == nil and ntags > 1 then
```

Comment:
Could you also change the comment above? "If no tags are visible (and we did not delete the last tag), try and view one". This would actually describe this line of code a bit better. (At first I was confused here)